go/libkb/socket_windows_test.go

```diff
@@ -83,32 +83,39 @@ func namedPipeClient(sendSocket Socket, t *testing.T) {
 }
 
 func TestWindowsPipeOwner(t *testing.T) {
-	t.Skip()
+
+	t.Skip("Skipping pipeowner test - doesn't work on CI, works fine locally")
```

Comment:
there's probably a way to know if you're on CI, and then you can just skip there.